### DISCUSSION TITLE
Detecting LED pulse rate

### DISCUSSION THREAD

**Discussion by sheminasalam** [jomjol/AI-on-the-edge-device#3033] [General]
Hi there. I have a digital 3 phase bi directional electric meter. I have been trying to add this to my home assistant system for some time. There are no exposed communication ports with this sensor. There is an led which blinks 2000 times for 1kWH of electricity consumption. I have tried using a pulse meter with esphome but since the LED is recessed into the body, the LDR sensor is not working as expected. Since it is a digital meter, it is difficult to implement this project as such. I wanted to if it would be possible to detect the led blinks instead of character recognition with this setup so as to give the parameter impulse/minute. This could be used to find power.

**Comment by caco3** [comment]
The AIOTED project has a rater slow process time (at least  few seconds per round).

So no, I don't think this is possible with this project. I think it would be easier to start a new project for this feature.
